.gitignore

```diff
@@ -14,3 +14,4 @@ hs_err_pid*
 # Gradle stuff
 build/
 .gradle/
+/.nb-gradle/
```

Comment:
The file has not changed. This should not be part of the commit.

Reply:
true

Reply:
Sorry for that. My mistake. Thanks for the explanation.